SOPT-iOS/Projects/Features/HomeFeature/Sources/HomeScene/Cells/DashBoard/DashBoardCardCVC.swift

```diff
@@ -13,6 +13,9 @@ import DSKit
 
 final class DashBoardCardCVC: UICollectionViewCell {
     
+    lazy var profileEditTap = profileEditView.gesture()
+    var cancelBag = CancelBag()
```

Comment:
```suggestion
    private(set) lazy var profileEditTap = profileEditView.gesture()
    private(set) var cancelBag = CancelBag()
```
두 프로퍼티 모두 외부에서 수정은 되지 않도록 `private(set)`을 붙여주면 좋을 것 같아요!